src/Mvc/Mvc.Core/test/Infrastructure/ValidationProblemDetailsJsonConverterTest.cs

```diff
@@ -4,6 +4,7 @@
 using System.Linq;
 using System.Text;
 using System.Text.Json;
+
```

Comment:
```suggestion
```

Reply:
using System.Text;
using System.Text.Json; 

These are not global usings in this project and they are required for the `System.Text.Encoding` and `System.Text.Json.Utf8JsonReader`

Reply:
However, I can remove `System.Linq` and `Xunit`